From 3b17bfadd58e4e75e5272d08e44835de9c17dffd Mon Sep 17 00:00:00 2001
From: carlosperate <carlosperate@users.noreply.github.com>
Date: Fri, 14 Apr 2017 20:13:53 +0100
Subject: [PATCH] Edit heap for vagrant build env

---
 source/microbit/mprun.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/microbit/mprun.c b/source/microbit/mprun.c
index 4c8acc4..8123ca9 100644
--- a/source/microbit/mprun.c
+++ b/source/microbit/mprun.c
@@ -99,7 +99,7 @@ void mp_run(void) {
     mp_stack_set_limit(1800); // stack is 2k
 
     // allocate the heap statically in the bss
-    static uint32_t heap[9712 / 4];
+    static uint32_t heap[9356 / 4];
     gc_init(heap, (uint8_t*)heap + sizeof(heap));
 
     /*
